setup.py

```diff
@@ -72,6 +69,8 @@
         'Programming Language :: Python :: 3.8',
         'Topic :: Scientific/Engineering :: Bio-Informatics',
     ],
-    use_scm_version={"local_scheme": "no-local-version"},
     setup_requires=['setuptools_scm'],
+
+    # 2.0.1 allows for a pip requirement formatted like "neurom>=2.0.0"
+    version="2.0.1.dev9",
```

Comment:
not sure what you want to do here

Reply:
Ah, that's my bad. When I did a tmp release of this branch, it got here. I'm removing this. Thank you.

Reply:
Ah no, actually it has been added by Benoit, I need to think on this.

Reply:
removed it